clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/spring/client/properties/CamundaClientProperties.java

```diff
@@ -183,6 +195,18 @@ public URI getRestAddress() {
   }
 
   public void setRestAddress(final URI restAddress) {
+    /*
```

Comment:
Move to Util class (CamundaClientConfigurationUtil?) in the java client and call the method (CamundaClientConfigurationUtil.validateRestAddress) from there.